packages/tools/custom-widgets-utils-internal/package.json

```diff
@@ -44,6 +44,7 @@
     "json-loader": "^0.5.7",
     "mendix-client": "^7.15.8",
     "mendix-widget-build-script": "^1.0.3",
+    "mini-css-extract-plugin": "^1.2.1",
```

Comment:
Was this just forgotten or is there more to this change?

Reply:
It was just being used in `custom-widgets-utils-internal` but wrongly added to the root and not in the package.